Updates for release 4.0.17 (#26)
minio/directpv-docs: content/installation/upgrade.md

DirectPV 4.1.x and later removed support for `PodSecurityPolicy`.
{{< /admonition >}}

## Version support

> Version **4.1.x** is the latest lineage of open source DirectPV CSI Driver.
> Most users should install or upgrade to the latest release in this lineage.
> However, if you require continued use of the deprecated `PodSecurityPolicy` feature, be sure to follow the upgrade guide carefully.
> 
> Version **4.0.x** lineage entered maintenance mode on January 1, 2025.
> There is no future development of this lineage expected, though it may continue to receive bug fix and security releases from time to time.
> 
> Versions older than 4.0.0 are no longer supported.
> 

Reviewer: I ... have never encountered the term "lineage" to refer to this kind of version association, and it does not seem common. (I couldn't find it anywhere else in our docs, either ...) So ... suggestion, happy to discuss further. I don't think we need any equivalent, though -- this isn't an uncommon thing with release versions ime ...

```suggestion
Version **4.1.x** is the most recent version of the open source DirectPV CSI Driver.
Most users should install or upgrade to the latest release in this lineage.
However, if you require continued use of the deprecated `PodSecurityPolicy` feature, be sure to follow the upgrade guide carefully.

Version **4.0.x** entered maintenance mode on January 1, 2025.
No future development of this version is expected. It may continue to receive bug fix and security releases from time to time.

Versions older than 4.0.0 are no longer supported.

```

Author: Did some other adjustments for this section 8using headings and the word "series" instead of "lineage".

Author: Let me know if the new changes address your concerns.

DirectPV 4.1.x and later removed support for `PodSecurityPolicy`.
{{< /admonition >}}

## Version support

### 4.1.x

Releases in the **4.1.x** series are the latest of the open source DirectPV CSI Driver.

Most users should install or upgrade to the latest release in the **4.1.x** series.

However, if you require continued use of the deprecated `PodSecurityPolicy` feature, be sure to follow the upgrade guide carefully.

### 4.0.x

The **4.0.x** series entered maintenance mode on January 1, 2025.

> There is no future development of this series expected.
> Releases in this series may appear from time to time to address bugs or security concerns.

Reviewer 2: ```suggestion
Maintenance may include limited bug or security resolution.
Users should plan to migrate to the 4.1.x series at the earliest opportunity.
```

Reviewer 2: OR

```suggestion
Bug or security issues may only be addressed by upgrading to the 4.1.x series.
```

Author: 👍
